exploring-ship-logbooks/word_count.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import math
+
+def word_count(data, columns, key_words):
+    """
+    Generates a list containing the number of counts of each key word in
+    each logbook entry.
+
+    Inputs:
+        - data: raw logbook data
```

Comment:
Good comment! The note is in a weird position though and also the punctuation is not consistent throughout the comment.